english/time-travel/0/5.md

```diff
@@ -0,0 +1,4 @@
+**You sit in the dark house alone and fall asleep. While you were sleep, a tiger enters your wood built house.**
```

Comment:
sleep - sleeping
you can use Grammarly for grammar check

Reply:
Thanks, I updated it. I don't know how to use Grammarly in markdown, so I will need to figure that out for the future.